src/Tizen.Multimedia/AudioManager/AudioDevice.cs

```diff
@@ -261,7 +292,10 @@ public uint GetSampleRate()
         }
 
         /// <summary>
-        /// Sets the device's 'avoid resampling' property.
+        /// Sets the 'avoid resampling' property for the audio device.
+        /// This property controls whether the device should avoid resampling audio data during playback.
+        /// Enabling this feature can help preserve audio quality by preventing alterations to audio that
+        /// may happen during format conversions.
```

Comment:
I guess "during playback" is more proper instead of "during format conversions".

Reply:
Okay